client/src/components/DataFiles/DataFilesModals/DataFilesPreviewModal.jsx

```diff
@@ -14,8 +32,17 @@ const DataFilesPreviewModal = () => {
   const hasError = error !== null;
   const previewUsingTextContent = !isLoading && !hasError && content !== null;
   const previewUsingHref = !isLoading && !hasError && !previewUsingTextContent;
+  const previewUsingBrainmap =
+    !isLoading &&
+    !hasError &&
+    params.path &&
+    (params.path.endsWith('.nii') || params.path.endsWith('.nii.gz'));
```

Comment:
Any way we could marry this to the `SUPPORTED_BRAINMAP_PREVIEW_EXTS` setting?

Reply:
The issue here is the `.nii.gz`. Our typical method of parsing file extensions doesn't work because the extension for `file.nii.gz` will be read as `.gz` which isn't helpful. I don't want to rewrite extension parsing in V2 because I can foresee it being a pain point when we merge this into the V3 branch, or vice versa.

Reply:
I'm not sure I follow. Since this is already a special case couldn't this be something like (psuedocode) any([ext for ext in SUPPORTED_BRAINMAP_PREVIEW_EXTS if params.path.endsWith(ext)])

Reply:
Fair enough, I forgot python had an `any` so I thought the list comprehension would be way grosser.